### PR TITLE
Feature/currency by location : replacing old PR

### PR DESCRIPTION
**Pull request by mahmed0715**
This is updated PR, which uses old currency rates api.

### CODE REVIEW

**Comment by mahmed0715** [comment]
replacing https://github.com/Plant-for-the-Planet-org/treecounter-app/pull/1572

**Comment by mahmed0715** [comment]
@norbertschuler this is a new PR which uses old currency api. No change in calculation @sagararyal

File: app/actions/fetchLocation.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          console.error(error);
+        });
+    }
+  };
+}
+
+export function fetchCurrencies() {
+  return dispatch => {
+    getExternalRequest({
+      endPoint: `https://openexchangerates.org/api/latest.json?app_id=${
```
Comment (norbertschuler) [inline app/actions/fetchLocation.js]: I don't understand this. In #1572 we changed using this endPoint directly in the client towards using a new API call `getRequest('public_currencies_get', { version: 'v1.3' })` developed by Jorgo to limit the number of API calls (as if every client calls it we would produce a lot more API requests as if our backend makes the request and caches it for all clients). I saw that in the backend code base - is that right @jmiridis? Now we don't want to tunnel this request through our backend any more again, do we?

File: app/config/index.js.dist [hunk 2]
```diff
@@ -22,5 +22,7 @@ export const context = {
   },
   ios:{
     appId: '1444740626'
-  }
+  },
+  locationApikKey: '1dfab408903c332b660f5e7cb90da06a',
+  currencyApiKey: 'd581a5406d694920aa493b9143a9e257'
```
Comment (norbertschuler) [inline app/config/index.js.dist]: If we move the currency API to the backend again, a API key does not need to be in the client as well.

Comment (mahmed0715) [reply app/config/index.js.dist]: true. removed.

File: app/actions/fetchLocation.js (hunk 1 above)
Comment (mahmed0715) [reply app/actions/fetchLocation.js]: sill we are using fetchcurrencies from actions/currencies.js 
now removed this one.

**Comment by mahmed0715** [comment]
@jmiridis please update the currency rates frequently, I see its 18 days old.

**Comment by mahmed0715** [comment]
@norbertschuler its clean now. please step forward.

**Comment by mahmed0715** [comment]
Lets merge this asap if good. because this will fix an issue reported by user https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1640. @norbertschuler

**Pull request merged**